website/pages/blog/graphiql-3/index.mdx

```diff
@@ -0,0 +1,212 @@
+project in general. Below is a list of various improvements that were made by me:
+
+1. Updated Cypress to v12 from v4 [#2893](https://github.com/graphql/graphiql/pull/2893).
+
+1. Avoid unnecessary renders component's tree with
+   [`eslint-plugin-react-usememo`](https://github.com/arthurgeron/eslint-plugin-react-usememo)
+   [#3124](https://github.com/graphql/graphiql/pull/3124).
+
+1. Updated all examples to use React 18 [#3185](https://github.com/graphql/graphiql/pull/3185).
+
+1. Excluded `"dependencies"` from bundle for `@graphiql/react`, `@graphiql/plugin-explorer` and
+   `@graphiql/plugin-code-exporter` [#3229](https://github.com/graphql/graphiql/pull/3228) and
+   [PR](https://github.com/graphql/graphiql/pull/3229).
+
+1. Removed unnecessary `<div />` wrappers [#3235](https://github.com/graphql/graphiql/pull/3235).
+
+1. Simplified Next.js and Vite examples with `monaco-graphql`
+   [#3140](https://github.com/graphql/graphiql/pull/3140) and
+   [#3281](https://github.com/graphql/graphiql/pull/3281).
+
+1. And
+   [more...](https://github.com/graphql/graphiql/pulls?q=is%3Apr+sort%3Aupdated-desc+author%3AB2o5T+is%3Aclosed)
+
+## GraphiQL 3
+
+Integration of `monaco-editor` and `monaco-graphql` always was a top priority, since I started
+maintaining this monorepo, but to achieve it, a stable release should be released first ☝️.
+
+So I decided to propose to release GraphiQL 3 without Reach UI and replace him with maintained
+libraries [Radix](https://radix-ui.com) and [Headless UI](https://headlessui.com) and to make
+another major release with `monaco-editor` bumped as GraphiQL 4.
+
+|    Version    |           What is inside            |
+| :-----------: | :---------------------------------: |
+|  GraphiQL 2   |        CodeMirror / Reach UI        |
+|  GraphiQL 3   |  CodeMirror / Radix / Headless UI   |
+| GraphiQL 4 🔜 | Monaco-Editor / Radix / Headless UI |
+
+<p className="mt-3 text-center italic">Table of comparison versions of GraphiQL</p>
+
+With all the above changes I also included a tab reorders feature:
+
+![Reorder tabs feature](./reorder-tabs.gif)
+
+{/* prettier-ignore */}
+<p className="mt-3 text-center italic">Tab reorders made by
+  [@mylesmmurphy](https://github.com/mylesmmurphy)
+  [#3111](https://github.com/graphql/graphiql/pull/3111)</p>
+
+Try GraphiQL 3 by installing it with your favourite package manager:
+
+```sh npm2yarn
+npm i graphql@3
+```
+
+## GraphiQL 4 🔜
+
+14 months ago the initial issue was created on the integration `monaco-editor` in GraphiQL.
+
+{/* prettier-ignore */}
+<Anchor
+  href="https://github.com/graphql/graphiql/issues/2326#issue-1208525253" className="mt-6 block"
+>![Initial comment on integration of monaco-editor in GraphiQL](./img_4.png)</Anchor>
+
+but no real progress in GraphiQL was made (except
+[`graphiql-prototype`](https://github.com/jonathanawesome/graphiql-prototype) by
+[@jonathanawesome](https://github.com/jonathanawesome)).
+
+<Callout type="warning">
```

Comment:
I would phrase it a bit different and less offensive
Maybe you can say that there was some initial work on Jon on a prototype outside of GraphiQL without integration